qodana.yaml

```
@@ -2,3 +2,7 @@ version: "1.0"
 linter: jetbrains/qodana-jvm
 profile:
   path: .idea/inspectionProfiles/Project_Quality.xml
+exclude:
+  - name: All
+    patterns:
+      - "**/gen/**"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)